### PR TITLE
change hover & color search

### PR DESCRIPTION
**Pull request by MatYouKy**
# Related issue
- change notifications hover and search color

# Changes
- change search color
- change hover

# Screenshots 

**Before**
Search
search before

Notification
notification before

**After**

Search
search after

Notification
notifications

### CODE REVIEW

File: src/styles/Colors.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 $error: #ff4d4dd5;
 $lightBlue: #c4cfdb;
 $linkedinBlue: #3f51b5;
+$linkedinLightBlue: #3f51b512;
```
Comment (MateuszLisowski) [inline src/styles/Colors.scss:4]: I think it should be #3f51b5 (from my research #3f51b512 doesnt exist)

File: src/components/Notification/Notification.scss [hunk 2]
```diff
@@ -4,12 +4,12 @@
 
 .notificationCard {
   display: flex;
-  margin: 5px;
+  padding: 10px;
   word-break: break-all;
-  transition: transform 100ms;
-
+  transition: background-color 200ms;
```
Comment (MateuszLisowski) [inline src/components/Notification/Notification.scss:9]: you should read https://www.creativebloq.com/advice/tips-for-css-animation and https://csstriggers.com/ what and how animate 

tldr your code is ok but could be better

File: src/components/Dropdown/Dropdown.scss [hunk 3]
```diff
@@ -29,7 +29,7 @@
     @media only screen and (min-width: $mobile) {
       max-width: 200px;
       right: 20px;
-      padding: 10px 20px;
+      padding: 10px 0px;
```
Comment (MateuszLisowski) [inline src/components/Dropdown/Dropdown.scss:32]:
```suggestion
      padding: 10px 0;
```

File: src/components/Dropdown/Dropdown.scss [hunk 4]
```diff
@@ -14,8 +14,8 @@
     z-index: $ZIndexDropdown;
     box-shadow: 0px 0px 8px 1px rgba(0, 0, 0, 0.3);
     position: absolute;
-    padding: 10px 20px;
-    min-width: 150px;
+    padding: 10px 0px;
```
Comment (MateuszLisowski) [inline src/components/Dropdown/Dropdown.scss:17]:
```suggestion
      padding: 10px 0;
```

**Review by MateuszLisowski — CHANGES_REQUESTED** [review]

**Comment by MateuszLisowski** [comment]
Please provide screenshot

File: src/styles/Colors.scss (hunk 1 above)
Comment (MatYouKy) [reply src/styles/Colors.scss:4]: The last 2 digits are transparent.

File: src/components/Dropdown/Dropdown.scss (hunk 3 above)
Comment (MateuszLisowski) [reply src/components/Dropdown/Dropdown.scss:32]: still occurs

**Review by MateuszLisowski — APPROVED** [review]

**Review by jakub-gosciniak — APPROVED** [review]
